capstone-ui/src/constants.js

```diff
@@ -227,4 +227,57 @@ const fipsStateCodes = {
   '56': 'wyoming',
 };
 
-export { options, unclickables, questionsWithOptions, parties, baseCandidates, partiesOrganized, FIPSfromStateName, fipsStateCodes };
\ No newline at end of file
+const presidentialStateWins = {
```

Comment:
This file is getting to large. Let's create a constants folder with different constants files. For example, StatesConstants.js could have `fipsStateCodes` & ` presidentialStateWins ` while CandidatesConstants can have `baseCandidates`